plugins/sources/s3.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+var env, s3, stream, util, client, bucket;
+
+env    = require('../../node_modules/image-resizer/src/config/environment_vars');
+s3     = require('aws-sdk').S3;
+stream = require('stream');
+util   = require('util');
+
+try {
+  // create an AWS S3 client with the config data
+  client = new s3({
+    accessKeyId: env.AWS_ACCESS_KEY_ID,
+    secretAccessKey: env.AWS_SECRET_ACCESS_KEY,
+    region: env.AWS_REGION
+  });
+  bucket = env.S3_BUCKET;
+} catch(e) {
+
```

Comment:
So, if it encounters an error it silently swallows an error condition and then continues execution without indicating any error has taken place or providing alteratives?